rust/src/ledger/public_key.rs

```diff
@@ -9,13 +9,30 @@ use serde::{Deserialize, Serialize};
 pub struct PublicKey(pub String);
 
 impl PublicKey {
+    pub const LEN: usize = 55;
+    pub const PREFIX: &'static str = "B62q";
```

Comment:
This seems suspicious. Do we know that this will always be the prefix?

Reply:
We do. We've been using them for a long time now. Just wanted to make them a little handier.